composer.json

```diff
@@ -16,18 +16,18 @@
     ],
     "require": {
         "php": ">=7.2.5",
-        "symfony/framework-bundle": "^5.4 || ^6.0 ",
-        "symfony/intl": "^5.4 || ^6.0",
-        "symfony/templating": "^5.4  || ^6.0",
-        "symfony/validator": "^5.4  || ^6.0",
-        "symfony/yaml": "^5.4  || ^6.0",
+        "symfony/framework-bundle": "^5.4 || ^6.0 || ^7.0 ",
```

Comment:
with the way the ci.yaml file is set up, you also need to add a section to have it actually test with symfony 7.